packages/vue3-vite/src/generators/vue3-vite/generator.ts

```diff
@@ -42,6 +44,7 @@ function normalizeOptions(
     projectDirectory
   );
   const parsedTags = parseTags(options.tags);
+  const useLocalAlias = (options.alias || PathAlias.Local) === PathAlias.Local;
```

Comment:
I have to write it like this because it seems that `options` will not read the `default` defined in `schema.json`.
It's probably because it is not designed to work that way. 😅

Reply:
I think I've been confused by this before. It does seem like you're right, the default from `schema.json` is only for letting the user know, so we need to manually ensure it matches the generator code.